### PR TITLE
Fix login page styling

### PR DESCRIPTION
**Pull request by Elkrival**
* Added background color to footer
* Added z-index due to the label from the text field having a zindex that overlays on top of the footer

Screenshot 2023-12-05 at 1 28 46 PM

### CODE REVIEW

**Comment by mikestone14** [comment]
@Elkrival looking at the screenshot we still haven't address the core issue of the page content being cut off.

**Comment by Elkrival** [comment]
> @Elkrival looking at the screenshot we still haven't address the core issue of the page content being cut off.

gotcha here are the updates [096fbd3](https://github.com/AMP-SCZ/dpdash/pull/510/commits/096fbd3669887b10ea2e33bfd976021e8698c429)

Screenshot 2023-12-06 at 10 20 02 AM
Screenshot 2023-12-06 at 10 24 15 AM

File: views/components/HeroCard/HeroCard.css [hunk 1]
```diff
@@ -10,12 +10,14 @@
   height: 80px;
 }
 
-@media screen and (min-width: 768px) {
+@media screen and (min-width: 900px) {
   .HeroCard_container {
     width: 565px;
+    padding-top: 162px;
```
Comment (JackVCurtis) [inline views/components/HeroCard/HeroCard.css]: How do we get to this number? It feels random/too specific but I may just be missing the idea.

File: views/components/HeroCard/HeroCard.css [hunk 2]
```diff
@@ -10,9 +10,10 @@
   height: 80px;
 }
 
-@media screen and (min-width: 768px) {
+@media screen and (min-width: 900px) {
   .HeroCard_container {
     width: 565px;
+    /* flex-grow: 1; */
```
Comment (JackVCurtis) [inline views/components/HeroCard/HeroCard.css:16]: Stray comment

File: views/layouts/HeroLayout/index.jsx [hunk 3]
```diff
@@ -1,29 +1,83 @@
...
-import './HeroLayout.css'
 
 const HeroLayout = () => {
   return (
-    <Box className="HeroLayout_container">
-      <Box className="HeroLayout_main">
-        <HeroCard />
-
-        <Box
+    <Box sx={{ height: '100vh' }}>
```
Comment (JackVCurtis) [inline views/layouts/HeroLayout/index.jsx:10]: I've got personal gripes with component-based grid systems but I guess this is the Material Way.

**Review by JackVCurtis — APPROVED** [review]
Looks good aside from the comment! Real glad we cleared up that positioning code, I think this is much more robust

**Pull request merged**